### PR TITLE
TECH-146: Create test configuration for "/vouchers/voucher_activation" endpoint

### PR DESCRIPTION
**Pull request by mgrochalskisoldevelo**
**Don't merge before TECH-145** : https://github.com/GovStackWorkingGroup/bb-payments/pull/6

## Description
Create test configuration for /vouchers/voucher_activation endpoint

TICKET: https://govstack-global.atlassian.net/browse/TECH-146

### CODE REVIEW

**Comment by mgrochalskisoldevelo** [comment]
@dborowiecki @dszafranek @conradsp @KarolinaKopacz 
I had to rebase from Karolina's branch because of critical changes in the project structure. 
That's why PR includes her commits, they should disappear after TECH-145 is merged.

Please review only my changes : https://github.com/GovStackWorkingGroup/bb-payments/pull/7/commits/03db9066b90717d4bf5318f27f38e9d68ce0b973

Please don't merge before TECH-145 : https://github.com/GovStackWorkingGroup/bb-payments/pull/6

File: test/features/support/voucher_activation.js
Comment (KarolinaKopacz) [inline test/features/support/voucher_activation.js:11]: For feature reference: `validBodyValue` is only used once, so no need to put it here as a constant. You can just write in line 36 as:
```
.withBody({
    "voucher_serial_number": 223,
    "Gov_Stack_BB": "example Gov_Stack_BB"
})
```
The same with `invalidSerialNumberBodyValue` and `invalidGovStackBBBodyValue`
:)

File: test/features/voucher_activation.feature
Comment (KarolinaKopacz) [inline test/features/voucher_activation.feature:21]: Do you mean the user is not able to activate a pre-activated voucher because of Gov Stack Building Block does not exist? If yes, please change the title and error message.
Zrzut ekranu 2023-01-9 o 12 51 46

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

File: test/features/support/voucher_activation.js
Comment (KarolinaKopacz) [inline test/features/support/voucher_activation.js:28]: Please fix the formatting

File: test/features/support/voucher_activation.js
Comment (KarolinaKopacz) [inline test/features/support/voucher_activation.js:62]: Please fix the formatting

File: test/features/support/voucher_activation.js
Comment (KarolinaKopacz) [inline test/features/support/voucher_activation.js:79]: Please fix the formatting

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

File: test/features/voucher_activation.feature
Comment (KarolinaKopacz) [reply test/features/voucher_activation.feature]: @mgrochalskisoldevelo

Comment (mgrochalskisoldevelo) [reply test/features/voucher_activation.feature]: @KarolinaKopacz yes, that's what i meant

https://github.com/GovStackWorkingGroup/bb-payments/pull/7/commits/823af1d2e95347ed8ffc773d87508f276b27fa84

image

Comment (KarolinaKopacz) [reply test/features/voucher_activation.feature]: I didn't see those changes, sorry :)

**Review by KarolinaKopacz — APPROVED** [review]

**Review by dborowiecki — APPROVED** [review]
Looks good for me.

**Review by oscarcorreia43 — APPROVED** [review]

**Review by dborowiecki — APPROVED** [review]
LGTM

**Pull request merged**
